### PR TITLE
main/imagemagick: upgrade to 7.0.8.49

### PR DESCRIPTION
**Pull request by J0WI**
Source url changed, because older releases are no longer available on imagemagick.org

### CODE REVIEW

File: main/imagemagick/APKBUILD [hunk 1]
```diff
@@ -16,7 +16,7 @@ makedepends="zlib-dev libpng-dev libjpeg-turbo-dev freetype-dev fontconfig-dev
 	libwebp-dev libxml2-dev librsvg-dev libx11-dev libxext-dev"
 checkdepends="freetype fontconfig ghostscript ghostscript-fonts lcms2 graphviz"
 subpackages="$pkgname-doc $pkgname-dev $pkgname-c++:_cxx $pkgname-libs"
-source="https://www.imagemagick.org/download/releases/ImageMagick-$_pkgver.tar.xz"
+source="https://github.com/ImageMagick/ImageMagick/archive/$_pkgver.tar.gz"
```
Comment (tcely) [inline main/imagemagick/APKBUILD]:
```suggestion
source="$pkgname-$pkgver::https://github.com/ImageMagick/ImageMagick/archive/$_pkgver.tar.gz"
```

File: main/imagemagick/APKBUILD [hunk 2]
```diff
@@ -85,4 +85,4 @@ _cxx() {
 	mv "$pkgdir"/usr/lib/libMagick++*.so.* "$subpkgdir"/usr/lib/
 }
 
-sha512sums="a18b9e78375ed1a8ab3526ead089dc39c4eed189d4aca0df87787c7ef6e4ee46cdc9a93a1fe1c2a65f0ac2f86c3651696ccd23eb82b6a96cad4c78cf8314bb96  ImageMagick-7.0.8-44.tar.xz"
+sha512sums="40ca3207b83c4c3cdf9c01320d27e98b3bceeb75c0ff256bbd0a2b29f185631801bda27c22c95e1834621ec6577e350adb690a4128198d2f57d7c8691e6562e5  7.0.8-47.tar.gz"
```
Comment (tcely) [inline main/imagemagick/APKBUILD]:
```suggestion
sha512sums="40ca3207b83c4c3cdf9c01320d27e98b3bceeb75c0ff256bbd0a2b29f185631801bda27c22c95e1834621ec6577e350adb690a4128198d2f57d7c8691e6562e5  imagemagick-7.0.8-47.tar.gz"
```

File: main/imagemagick/APKBUILD [hunk 3]
```diff
@@ -30,7 +30,6 @@ builddir="$srcdir/ImageMagick-${_pkgver}"
 #     - CVE-2019-10650
 
 build() {
-	cd "$builddir"
```
Comment (tcely) [inline main/imagemagick/APKBUILD]: Keep `cd` line.

File: main/imagemagick/APKBUILD [hunk 4]
```diff
@@ -59,12 +58,10 @@ build() {
 }
 
 check() {
-	cd "$builddir"
```
Comment (tcely) [inline main/imagemagick/APKBUILD]: Keep `cd` line.

File: main/imagemagick/APKBUILD [hunk 5]
```diff
@@ -59,12 +58,10 @@ build() {
...
 	make check
 }
 
 package() {
-	cd "$builddir"
```
Comment (tcely) [inline main/imagemagick/APKBUILD]: Keep `cd` line.

**Comment by tcely** [comment]
Check is failing for some architectures:

- `x86`: FAIL: `Magick++/tests/tests.tap 3`
- `aarch64`: FAIL: `tests/validate-formats-map.tap 1`
- `armv7`: FAIL: `tests/validate-compare.tap 1`

File: main/imagemagick/APKBUILD (hunk 5 above)
Comment (J0WI) [reply main/imagemagick/APKBUILD]: Done

**Comment by J0WI** [comment]
Tests result may depend on floating point handling of CPU.

**Comment by J0WI** [comment]
97eebdc34c933fc99db17f4204b0204bc58b1b52

**Pull request closed without merging**